src/database/migrations/20190907215523-update-trip-table.js

```diff
@@ -0,0 +1,53 @@
+export default {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('TripRequests', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    origin: {
+      type: Sequelize.STRING,
+      allowNull: false
+    },
+    destination: {
+      type: Sequelize.STRING,
+      allowNull: false
+    },
+    reason: {
+      type: Sequelize.STRING,
+      allowNull: false
+    },
+    departureDate: {
+      type: Sequelize.DATEONLY,
+      allowNull: false
+    },
+    returnDate: {
+      type: Sequelize.DATEONLY
+    },
+    type: {
+      type: Sequelize.STRING,
+      allowNull: false
+    },
+    status: {
+      type: Sequelize.STRING,
+      allowNull: false
+    },
+    userId: {
+      type: Sequelize.INTEGER,
+      allowNull: false
+    },
+    accomodationId: {
+      type: Sequelize.INTEGER
+    },
+    createdAt: {
+      allowNull: false,
+      type: Sequelize.DATE
+    },
+    updatedAt: {
+      allowNull: false,
+      type: Sequelize.DATE
+    }
+  }),
+  down: (queryInterface) => queryInterface.dropTable('TripRequests')
+};
```

Comment:
+1 to @joaquinto 's comment. I'm not sure we need the model. Since we already have a model `/src/database/migrations/20190829084157-create-trip-request.js`. Reference it an try to refactor.

Reply:
ok, will do that